spec/validations/svg_spec.rb

```diff
@@ -100,7 +110,7 @@
       let(:markup) { markup }
       it 'sanitises questionable markup and notifies the user that the offending svgs were modified' do
         errors = ZendeskAppsSupport::Validations::Svg.call(package)
-        expect(IO).to have_received(:write).with(svg.relative_path, clean_markup)
+        expect(IO).to have_received(:write).with(svg.relative_path, clean_markup[0])
```

Comment:
It doesn't seem clear that the first `clean_markup` is a sanitised version of these. Could we redefine it here to be explicit?